sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/AbstractAdoDao.java

```diff
@@ -1315,4 +1316,24 @@ public long countOf() {
 	public ConnectionSource getConnectionSource() {
 		return dao.getConnectionSource();
 	}
+
+	// dao utilities
+
+	protected void addDateFromCriteria(List<Where<Immunization, Long>> whereStatements, Where<Immunization, Long> where, Date dateFrom, String date) throws SQLException {
```

Comment:
Change Immunization references to AbstractDomainObject (since we're in a generic Dao) for all three methods